invite_user.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
+    <title>Platform One SSO</title>
+    <link rel="icon" type="image/png" sizes="502x498" href="img/babyyodaCrop.png">
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/styles.min.css">
+</head>
+<body id="page-top">
+    <nav class="navbar navbar-light navbar-expand-lg fixed-top" id="mainNav">
+        <div class="container"><a class="navbar-brand js-scroll-trigger" href="/p1sso/index.html"
+                style="color: #52a1fd;">PLATFORM ONE SSO</a><button data-toggle="collapse"
+                data-target="#navbarResponsive" class="navbar-toggler float-right" aria-controls="navbarResponsive"
+                aria-expanded="false" aria-label="Toggle navigation"><i class="fa fa-bars"></i></button>
+            <div class="collapse navbar-collapse" id="navbarResponsive"
+                style="margin: 0 auto;margin-left: 0px;padding: auto;margin-top: 0px;margin-right: 0px;margin-bottom: 0px;width: auto;min-width: 0px;max-width: none;height: auto;min-height: 0px;padding-left: 10px;">
+              
+            </div>
+        </div>
+    </nav>
```

Comment:
We should get rid of this entire `nav` block since there is no actual navigation. Also matches the create account page.